### ISSUE TITLE
"No theme package"

### ISSUE THREAD

**Issue by Libra983** [FaBjE/HiBy-Theme-MaterialDesign#7]
R3 Pro Saber with firmware 2.2 (downloaded from Hiby).

Getting message "No package theme" after clicking "use custom themes". Theme package is in root folder. 

If I'm missing a step, please advise.

**Comment by FaBjE** [comment]
The theme file must be placed in the "theme" folder you create on the root of your SD card

 _Place MaterialDesign.t file in "theme" folder._

**Comment by Libra983** [comment]
Yes. Material design.t is placed in a "theme" folder as per instructions. But I still get the message.

**Comment by FaBjE** [comment]
Could you try rebooting the player after placing the theme file?

What is the full path on your SD card to the theme file?
What is your exact player device type?

**Comment by Libra983** [comment]
Yes. I have rebooted after placing the file a couple of times and no dice. Same issue. I will check the file path when I get home from work. 
The device is R3 pro saber with the stick 2.2 firmware off the Hiby server. When I check on about , says device was built February 2023 

Yahoo Mail: Search, Organize, Conquer 
 
  On Tue, Mar 19, 2024 at 9:15 AM, ***@***.***> wrote:   


Could you try rebooting the player after placing the theme file?

What is the full path on your SD card to the theme file?
What is your exact player device type?

**Comment by Libra983** [comment]
This is the path

Yahoo Mail: Search, Organize, Conquer 
 
  On Tue, Mar 19, 2024 at 9:26 AM, ***@***.******@***.***> wrote:   Yes. I have rebooted after placing the file a couple of times and no dice. Same issue. I will check the file path when I get home from work. 
The device is R3 pro saber with the stick 2.2 firmware off the Hiby server. When I check on about , says device was built February 2023 

Yahoo Mail: Search, Organize, Conquer 
 
  On Tue, Mar 19, 2024 at 9:15 AM, ***@***.***> wrote:   


Could you try rebooting the player after placing the theme file?

What is the full path on your SD card to the theme file?
What is your exact player device type?
